src/core/connectors.js

```diff
@@ -293,7 +293,7 @@ define(function() {
 
 	{
 		label: 'Xiami',
-		matches: ['*://www.xiami.com/play*'],
+		matches: ['*://www.xiami.com/*'],
```

Comment:
need to consider the case no slash at the end? e.g `https://www.xiami.com`

Reply:
Should match since `*` is wildcard